workspace/welcome-to-xod/014-internet/patch.xodp

```diff
@@ -0,0 +1,128 @@
+{
+  "comments": [
+    {
+      "content": "# Internet\n\nThere is a large class of tasks, called Internet-Of-Things when the hardware needs an Internet connection to send or receive data from some server. For example, log the temperature and moisture for smart greenhouses, getting the current time, weather, work with XOD Cloud or other APIs, and so on.\n\nIn this chapter, we get the current date and time from the XOD Cloud API.\n\nFor this, there is the `xod-cloud/basics/now` node, which sends a request to the XOD Cloud, parses, and outputs the response as `datetime` type value.\n\nThis node has an input pin `INET`. This pin has to be connected with an internet provider. The Internet can be provided\n- with a microcontroller itself\n  `xod-dev/esp8266-mcu/connect`,\n- with an external module\n  `xod-dev/esp8266/connect`,\n- with a shield\n  `xod-dev/w5500/ethernt-shield`,\n- *from the PC*\n  `xod/debug/tethering-inet`.\n  It works either in simulation or debug mode.",
```

Comment:
The list of internet providers is potentially stale. Also, at this point a reader has not touched hardware yet. So, suggest shortening the list to something like:

The Internet can be provided:

- by the controller board (such as ESP8266)
- by an extension module (such as W5500 Ethernet Shield)
- by **your PC**. This trick is known as Internet tethering and works in simulation and debug modes